eZ/Publish/API/Repository/Exceptions/ContentFieldValidationException.php

```diff
@@ -13,7 +13,7 @@
 /**
  * This Exception is thrown on create or update content one or more given fields are not valid.
  */
-abstract class ContentFieldValidationException extends ForbiddenException
+abstract class ContentFieldValidationException extends ContentValidationException
```

Comment:
afaik it is merely a bug fix, some cases that used to throw `ContentValidationException` will instead now throw ContentFieldValidationException, this can from vendor perspective be viewed as bug fix as it was quite frankly using the wrong exception. But other cases stays as is, so if code is catching both *(like it should)*, then all is fine\*.

\* Assuming we revert the change this thread is starting on, as if you are catching both (as you should), you'll basically get unpredictable results after this change.

So 1) the lesser evil is that some cases as of 6.5 uses ContentFieldValidationException, we shall document it in bc notes so people are aware. 2) yes we can, it's a but fix, we don't change api doc here, both expectations still prevails, and `ContentFieldValidationException` continues to be the exception with additional errors, now also covering empty field errors.


does this compile? :)